frontend/src/i18n/locales/de/glossary.json

```diff
@@ -0,0 +1,154 @@
+{
+  "Cluster": "Cluster",
+  "Version": "Version",
+  "Memory": "Speicher",
+  "CPU": "CPU",
+  "Pods": "Pods",
+  "Events": "Events",
+  "Warnings": "Warnungen",
+  "Type": "Typ",
+  "Reason": "Grund",
+  "Namespaces": "Namespace",
+  "Namespace": "Namespace",
+  "toggle field visibility": "Feldsichtbarkeit umschalten",
+  "Condition": "Zustand",
+  "Status": "Status",
+  "Last Transition": "Letzte Transition",
+  "Last Update": "Letzte Aktualisierung",
+  "Container ID": "Container-ID",
+  "Image Pull Policy": "Image Pull Policy",
+  "Image": "Image",
+  "Args": "Args",
+  "Command": "Befehl",
+  "Environment": "Umwelt",
+  "Liveness Probes": "Lebensfähigkeitsproben",
+  "Ports": "Häfen",
+  "Volume Mounts": "Volumenhalterungen",
+  "Containers": "Container",
+  "Container Spec": "Container-Spezifikation",
+  "Conditions": "Bedingungen",
+  "tabs": "Registerkarten",
+  "Container": "Container",
+  "Config Maps": "Config Maps",
+  "Scope": "Scope",
+  "Subresources": "Sub-Ressourcen",
+  "Plural": "Plural",
+  "Singular": "Singular",
+  "Kind": "Art",
+  "List Kind": "Listen-Typ",
+  "Served": "Ausgeliefert",
+  "Storage": "Speicher",
+  "Objects": "Objekte",
+  "Schedule": "Planung",
+  "Suspend": "Anhalten",
+  "Starting deadline": "Beginn der Frist",
+  "Last Schedule": "Letzte Ausführung",
+  "Cron Jobs": "Cron Jobs",
+  "Tolerations": "Toleranzen",
+  "Update Strategy": "Strategie aktualisieren",
+  "Selector": "Selektor",
+  "Node Selector": "Knotenwähler",
+  "Daemon Sets": "Daemon Sets",
+  "Deployments": "Deployments",
+  "Replicas": "Replikate",
+  "Endpoint": "Endpunkt",
+  "Subsets": "Untergruppen",
+  "IP": "IP",
+  "Hostname": "Hostname",
+  "Target": "Ziel",
+  "Port": "Port",
+  "Protocol": "Protokoll",
+  "Endpoints": "Endpoints",
+  "Horizontal Pod Autoscalers": "Horizontal Pod Autoscalers",
+  "Rules": "Regeln",
+  "Host": "Host",
+  "Path": "Pfad",
+  "Service": "Service",
+  "Ingresses": "Ingresses",
+  "Hosts": "Hosts",
+  "Jobs": "Jobs",
+  "Completions": "Abschlüsse",
+  "Pod CIDR": "CIDR des Pods",
+  "Uptime": "Betriebszeit",
+  "System Info": "System-Informationen",
+  "Architecture": "Architektur",
+  "Boot ID": "Boot ID",
+  "System UUID": "UUID des Systems",
+  "OS": "OS",
+  "Kernel Version": "Kernel-Version",
+  "Machine ID": "Maschinen-ID",
+  "Kube Proxy Version": "Kube Proxy Version",
+  "Kubelet Version": "Kubelet-Version",
+  "Container Runtime Version": "Container-Laufzeitversion",
```

Comment:
Probably leave as Runtime? (Laufzeit would be "running time" - I think the "runtime" concept is probably not translated)